SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetricsDisplay.cpp

```diff
@@ -181,10 +194,40 @@ void ASpatialMetricsDisplay::Tick(float DeltaSeconds)
 
 	const USpatialMetrics& Metrics = *SpatialNetDriver->SpatialMetrics;
 
-	FWorkerStats Stats;
+	FWorkerStats Stats{};
 	Stats.WorkerName = SpatialNetDriver->Connection->GetWorkerId().Left(WorkerNameMaxLength).ToLower();
 	Stats.AverageFPS = Metrics.GetAverageFPS();
 	Stats.WorkerLoad = Metrics.GetWorkerLoad();
 
+#if USE_SERVER_PERF_COUNTERS
+	float MovementCorrectionsPerSecond = 0.f;
+	int32 NumServerMoveCorrections = 0;
+	float WorldTime = GetWorld()->GetTimeSeconds();
+	NumServerMoveCorrections = PerfCountersGet(TEXT("NumServerMoveCorrections"), NumServerMoveCorrections);
+	MovementCorrectionRecord OldestRecord;
+	if (MovementCorrectionRecords.Peek(OldestRecord))
+	{
+		float WorldTimeDelta = OldestRecord.Time - WorldTime;
```

Comment:
Do I need another coffee or is this backwards? `WorldTime` will grow upwards and this will always be negative?

Reply:
🤦‍♂